recipes/onnxruntime/all/conanfile.py

```diff
@@ -27,11 +27,13 @@ class OnnxRuntimeConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "with_xnnpack": [True, False],
+        "use_cuda": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "with_xnnpack": False,
+        "use_cuda": False,
```

Comment:
previous option is called `with_XX`, should we use the same naming across all the recipe?

Reply:
Good catch. I see that a bunch of other recipes use the option `with_cuda`. It would be good to change it to `with_cuda` so that it can easily be enabled across multiple recipes.

Reply:
There is a recommendation for option names: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#recommended-names